composer.json

```diff
@@ -9,28 +9,31 @@
     ],
     "license": "MIT",
     "require": {
-        "php": "^7.3",
+        "php": "^7.2.5",
         "ext-json": "*",
         "ext-simplexml": "*",
+        "fideloper/proxy": "^4.2",
+        "fruitcake/laravel-cors": "^1.0",
         "bensampo/laravel-enum": "^1.26",
+        "guzzlehttp/guzzle": "^6.3",
         "bonroyage/oauth": "1.*",
-        "laravel/framework": "^6.8",
-        "laravel/passport": "^7.3",
-        "laravel/tinker": "^1.0",
+        "laravel/framework": "^7.0",
+        "laravel/passport": "^8.4",
+        "laravel/tinker": "^2.0",
         "mll-lab/graphql-php-scalars": "^2.1",
         "nuwave/lighthouse": "^4.7",
-        "vatsimuk/auth-package": "dev-master"
+        "vatsimuk/auth-package": "dev-master",
+        "laravel/ui": "^2.0"
```

Comment:
Can we avoid the usage of this?

Reply:
Once ready for production, and auth-package v1 released yes, but seeing as we want to be using the most current version for now and it's locked in the lock file anyway, I don't see a problem with this while it is in development